elements/modal.js

```diff
@@ -1,6 +1,122 @@
 const widget = require('cache-element/widget')
 const Modal = require('../lib/modal-element')
 const html = require('choo/html')
+const css = require('sheetify')
+
+css('dat-colors')
+
+const prefix = css`
+  :host {
+    --input-height: 2rem;
+    --icon-height: 1.2rem;
+    --button-width: 2rem;
+    box-shadow: 0 1.2rem 2.4rem rgba(0,0,0,.5);
+    position: relative;
+  }
+  .exit {
```

Comment:
Should we prefix these with `:host` also? Could prevent possible conflicts; though not fussed about it tho hah